src/FffGcodeWriter.cpp

```diff
@@ -1634,35 +1634,36 @@ bool FffGcodeWriter::processSingleLayerInfill(const SliceDataStorage& storage, L
 
             infill_below_skin.removeSmallAreas(min_area, remove_small_holes_from_infill_below_skin);
 
-            if (infill_below_skin.size())
+            if (!infill_below_skin.empty())
             {
                 // need to take skin/infill overlap that was added in SkinInfillAreaComputation::generateInfill() into account
                 const coord_t infill_skin_overlap = mesh.settings.get<coord_t>((part.insets.size() > 1) ? "wall_line_width_x" : "wall_line_width_0") / 2;
 
-                if (infill_below_skin.offset(-(infill_skin_overlap + tiny_infill_offset)).size())
+                if (!infill_below_skin.offset(-(infill_skin_overlap + tiny_infill_offset)).empty())
                 {
                     // there is infill below skin, is there also infill that isn't below skin?
                     Polygons infill_not_below_skin = in_outline.difference(infill_below_skin);
                     infill_not_below_skin.removeSmallAreas(min_area);
 
-                    if (infill_not_below_skin.size())
+                    if (!infill_not_below_skin.empty())
                     {
                         constexpr Polygons* perimeter_gaps = nullptr;
                         constexpr bool connected_zigzags = false;
                         constexpr bool use_endpieces = false;
                         constexpr bool skip_some_zags = false;
                         constexpr int zag_skip_count = 0;
+                        constexpr coord_t outline_offset = 0;
+                        const size_t min_wall_line_count = std::max(static_cast<size_t>(1), wall_line_count);
```

Comment:
For literals, you can use `1L`, `1UL` or `1LL` and the like.

Reply:
You're absolutely correct, I don't know where my mind was at the time. Will change those `static_cast`s